src/lib/github/v4/apiRequestV4.ts

```diff
@@ -33,6 +34,13 @@ export async function apiRequestV4<DataResponse>({
   query: DocumentNode;
   variables?: Variables;
 }) {
+  const gqlQueryName = getQueryName(query);
+  const span = apm.startSpan(`GraphQL: ${gqlQueryName}`);
+  span?.setType('external', 'graphql', 'query');
```

Comment:
If you like, these can be specified in the `startSpan` call. E.g.:

```suggestion
  const span = apm.startSpan(`GraphQL: ${gqlQueryName}`, 'external', 'graphql', 'query');
```

Reply:
Nice, I'll try that!